python/pylibcugraph/pylibcugraph/utils.pyx

```diff
@@ -122,19 +122,18 @@ cdef get_numpy_type_from_c_type(data_type_id_t c_type):
         raise RuntimeError("Internal error: got invalid data type enum value "
                            f"from C: {c_type}")
 
-
-cdef get_c_type_from_numpy_type(numpy_type):
-    if numpy_type == numpy.int32:
+cdef get_c_type_from_numpy_cupy_type(numpy_cupy_type):
```

Comment:
I wouldn't rename this since `cupy` just uses `numpy` dtypes:
```
>>> import numpy as np
>>> import cupy as cp
>>> np.int32
<class 'numpy.int32'>
>>> cp.int32
<class 'numpy.int32'>
```

Reply:
Right, I made those changes prior to our sync. That should be fixed in the next commit